common/src/main/java/net/neoforged/gradle/common/interfaceinjection/InterfaceInjectionPublishing.java

```diff
@@ -0,0 +1,56 @@
+package net.neoforged.gradle.common.interfaceinjection;
+
+import net.neoforged.gradle.dsl.common.extensions.InterfaceInjections;
+import org.gradle.api.Action;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.attributes.AttributeContainer;
+import org.gradle.api.attributes.Category;
+import org.gradle.api.component.AdhocComponentWithVariants;
+
+public class InterfaceInjectionPublishing {
+
+    public static final String INTERFACE_INJECTION_ELEMENTS_CONFIGURATION = "InterfaceInjectionElements";
+    public static final String INTERFACE_INJECTION_API_CONFIGURATION = "InterfaceInjectionApi";
+    public static final String INTERFACE_INJECTION_CONFIGURATION = "InterfaceInjection";
+    public static final String INTERFACE_INJECTION_CATEGORY = "InterfaceInjection";
```

Comment:
This should probably be lowercase, to match both MDG, which uses `interfaceinjection`, and gradle itself, for which all category attribute values are lowercase.